R/conversion.R

```diff
@@ -91,12 +91,9 @@ convert <- function(text,
 
   # Define Pattern for latex code
 
-  unicode_latex$int <- as.integer(as.hexmode(unicode_latex$unicode))
-  char_latex <- ifelse(unicode_latex$int <= 255 & unicode_latex$int != 177, unicode_latex$chr,
-    ifelse(unicode_latex$int < 32768,
-      paste0("\\uc1\\u", unicode_latex$int, "*"),
-      paste0("\\uc1\\u", unicode_latex$int - 65536, "*")
-    )
+  unicode_int <- as.integer(as.hexmode(unicode_latex$unicode))
+  char_latex <- ifelse(unicode_int <= 255 & unicode_int != 177, unicode_latex$chr,
+    sprintf("\\uc1\\u%d*", unicode_int - if (unicode_int < 32768) 0 else 65536)
```

Comment:
`unicode_int` is a vector of integers. how about 
```suggestion
    sprintf("\\uc1\\u%d*", unicode_int - ifelse (unicode_int < 32768, 0, 65536))
```

Reply:
BTW, my "ninja code" is vectorized, though :)